Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -14,7 +14,9 @@
 using DafnyServer.CounterexampleGeneration;
 using System.Runtime.InteropServices;
 using System.Text.RegularExpressions;
+using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Dafny.LanguageServer.Workspace;
```

Comment:
Maybe we should moves this to a more neutral place, if using the simple threading API really doesn't suffice.